barretenberg/cpp/src/barretenberg/vm/avm/trace/helper.cpp

```diff
@@ -1,126 +1,65 @@
 #include "barretenberg/vm/avm/trace/helper.hpp"
 
+#include <algorithm>
 #include <cassert>
 
 #include "barretenberg/vm/avm/trace/mem_trace.hpp"
 
 namespace bb::avm_trace {
 
-/**
- * @brief Routine to log some slice of a trace of the AVM. Used to debug or in some unit tests.
- *
- * @param trace The whole trace for AVM as a vector of rows.
- * @param beg The index of the beginning of the slice. (included)
- * @param end The index of the end of the slice (not included).
- */
-void log_avm_trace([[maybe_unused]] std::vector<Row> const& trace,
```

Comment:
I agree that a rework of this logging mechanism was overdue. However, I would like to easily see logs without uploading a file to Google sheet. Is a "less" command over the newly generated CSV readable?

Reply:
Great. Next time I will try to use Vscode with a CSV plugin.